src/Manifest/Manifest.router.ts

```diff
@@ -83,6 +92,26 @@ export class ManifestRouter extends Router {
     }
   }
 
+  async getProjectManifestByCoordinates(req: AuthRequest) {
+    const created_location = server.extractFromReq(req, 'land')
+    const eth_address = req.auth.ethAddress
+
+    const project : any = await Project.findOne({
```

Comment:
This works! but we can type it better. There are two options:
- You could replace `any` here with the actual type, wich would be `ProjectAttributes | undefined` (from `/Project`)
- Or, better still, use the generic on `findOne` like so: `Project.findOne<ProjectAttributes>(...)`